backend/routes/forgotPassword.js

```diff
@@ -1,59 +1,59 @@
-// const crypto = require("crypto");
-// const nodemailer = require("nodemailer");
-// const express = require("express");
-// const router = express.Router();
-// const User = require("../models/User");
+const crypto = require("crypto");
+const nodemailer = require("nodemailer");
```

Comment:
I'm not sure if this is the right library to use what do you think? @bogdanned

Reply:
for a first version `nodemailer` is a great choice